.github/workflows/build-and-deploy.yml

```diff
@@ -521,6 +528,47 @@ jobs:
           custom-secret-0: ${{ secrets.AUR_USERNAME }}
           custom-secret-1: ${{ secrets.AUR_EMAIL }}
           custom-secret-2: ${{ secrets.AUR_SSH_PRIVATE_KEY }}
+
+  # Completes the release process for winget
+  release-winget:
+    needs:
+      - ci-preparation
+      - release
+    concurrency:
+      group: release-winget-${{ needs.ci-preparation.outputs.build-ref }}
+      cancel-in-progress: false
+    runs-on: windows-2022
+    env:
+      packageID: "Unibo.Alchemist"
+    if: >-
+      !github.event.repository.fork
+      && (
+        github.event_name != 'pull_request'
+        || github.event.pull_request.head.repo.full_name == github.repository
+      )
+      && fromJSON(needs.release.outputs.winget-deploy)
+    steps:
+      - name: Install winget
+        uses: Cyberboss/install-winget@v1
+      - name: Install wingetcreate
+        shell: pwsh
+        run: Invoke-WebRequest https://aka.ms/wingetcreate/latest -OutFile wingetcreate.exe
+      - name: Check package availability
+        id: check
+        shell: pwsh
+        continue-on-error: true
```

Comment:
why continue on error?

Reply:
it seems to be that in case of error the subsequent step would be skipped anyway

Reply:
I used it as a workaround to avoid the failure of the job the first time the PR gets released (because the winget package is not published yet)

Reply:
After thinking about it some more, I believe it can be removed since the release job runs successfully, and even if winget-release fails, the new version is still released, right?

Reply:
The version would be released and then fail, the CI will fail. How to we repair it? Do we have to perform a by-hand release first?

Reply:
This PR brings the fix to the MSI package explained before. So in order to release by hand with a correct MSI package, we need first to distribute the package somewhere and put its url in the winget manifest. 

We can split this PR into two separate parts. The first part will address the package fix and the implementation of the test-msi installation. Once this part is completed and released, the next step would be to manually proceed with the winget-release. After that, we can merge this PR correctly.

Reply:
@Zimbrando can you generate the MSI yourself and release it manually, then we merge this?
I don't want to split, everything looks nice

Reply:
let me know when it's done, I'll merge this and then a breaking change PR that I created already. We'll soon know if this works

Reply:
Looks like it's been merged, should we go forward with this one?

Reply:
Yes, make sure `secrets.GIT_AUTH_TOKEN` holds a github authentication token, then we can merge this.

Reply:
Can't we simply use `github.token`? Which permissions does it need?